Neos.Neos/Classes/Domain/Service/UserService.php

```diff
@@ -303,8 +310,9 @@ public function emitUserCreated(User $user)
      */
     public function deleteUser(User $user)
     {
+        /** @var Account $account */
         foreach ($user->getAccounts() as $account) {
-            $this->deletePersonalWorkspace($account->getAccountIdentifier());
+            $this->deletePersonalWorkspace($user);
```

Comment:
This will invoke the same code for each account. This line should probably be moved out of the `foreach` loop

Reply:
lol, + 💯